orchestra/todos/views.py

```diff
@@ -47,8 +47,7 @@ def update_todos_from_todolist_template(request):
     try:
         add_todolist_template(todolist_template_slug, project_id, step_slug)
         todos = Todo.objects.filter(
-            project__id=project_id,
-            step__slug=step_slug).order_by('-created_at')
+            project__id=project_id).order_by('-created_at')
```

Comment:
why this change?

Reply:
There was a bug in code where we were showing only the todos associated with the step role for which the todolist template was added (will add to PR description). I can show it to you live.